static/js/activate-profile.js

```diff
@@ -25,3 +25,21 @@ $(function () {
   );
 });
 
+// Example starter JavaScript for disabling form submissions if there are invalid fields
+(function() {
+  'use strict';
+  window.addEventListener('load', function() {
```

Comment:
It might make sense to move this inside the `$(function) () {...}` block above as this is equivalent to the ready event (so we could drop the load listener).

I appreciate this is also the bootstrap-provided code, but since we're using jQuery we could also jqueryify it a bit (although very much  a matter of preference).
```
  $('form').on('submit', function(event) {
    $('.needs-validation').each(function() {
      if (this.checkValidity() === false) {
        event.preventDefault();
        event.stopPropagation();
      }
      $(this).addClass('was-validated');
    });
  });
```

Reply:
I'll leave it as is for now, mainly as the simple copy & paste of your code didn't seem to work for me 😂